samples/azure-quantum/chemistry/Hamiltonian/Approximate Energy using Jordan-Wigner transformation.ipynb

```diff
@@ -0,0 +1,813 @@
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Preparing Q# environment...\n"
+     ]
+    }
+   ],
+   "source": [
+    "# First, import qsharp and qsharp.azure to be able to compile and submit the quantum program.\n",
+    "import qsharp\n",
+    "import qsharp.azure"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9efb9be7",
+   "metadata": {},
+   "source": [
+    "### 1. Load H2 molecule data\n",
+    "\n",
+    "The pre-generated Broombridge file included in this sample contains the details on the molecule's Hamiltonian. A Hamiltonian is an operator that calculates the energy of a molecule by acting on a qubit register that represents the molecule's quantum state by Jordan-Wigner encoding. If we prepare the molecule in the ground state, then we know that applying the Hamiltonian we will calculate the ground state energy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "601fb1cb",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-08T20:46:20.76547-07:00\",\"IsCompleted\":false,\"Description\":\"Adding package microsoft.quantum.chemistry.jupyter\",\"Subtask\":null}",
+      "text/plain": [
+       "Adding package microsoft.quantum.chemistry.jupyter."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qdk.chemistry.broombridge import load_and_encode"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "f11f81ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "encoded_data_h2 = load_and_encode(\"../data/broombridge/hydrogen_0.2.yaml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ff4fc44",
+   "metadata": {},
+   "source": [
+    "This contains a tuple of the number of qubits, the fermionic Hamiltonian term coefficients and the energy offset.\n",
+    "\n",
+    "The index of each list in the second tuple is will be referred to as the \"term type\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "38c84985",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_qubits, fermion_terms, _, energy_offset = encoded_data_h2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57757a37",
+   "metadata": {},
+   "source": [
+    "Reload the workspace to load the Q# program for import into Python."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "94dd60f9",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/json": "{\"LastUpdated\":\"2021-05-08T20:46:28.063571-07:00\",\"IsCompleted\":false,\"Description\":\"Reloading workspace\",\"Subtask\":null}",
+      "text/plain": [
+       "Reloading workspace."
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "qsharp.reload()"
```

Comment:
For the demo, I combined all imports and reloads in one cell, since each of them is not particularly interesting on their own, just a prelude.